packaging/nntrainer.spec

```diff
@@ -79,6 +90,35 @@ Summary:	NNTrainer UnitTest Coverage Analysis Result
 HTML pages of lcov results of NNTrainer generated during rpmbuild
 %endif
 
+%if %{with tizen}
+%package -n capi-nntrainer
+Summary:         Tizen Native API for NNTrainer
+Group:           Multimedia/Framework
+Requires:        %{name}=%{version}-%{release}
+%description -n capi-nntrainer
+Tizen Native API wrapper for NNTrainer.
+You can trian neural networks efficiently.
+
+%post -n capi-nntrainer -p /sbin/ldconfig
+%postrun -n capi-nntrainer -p /sbin/ldconfig
+
+%package -n capi-nntrainer-devel
+Summary:         Tizen Native API Devel Kit for NNTrainer
+Group:           Multimedia/Framework
+Requires:        capi-nntrainer = %{version}-%{release}
+Requires:        capi-ml-common-devel
+%description -n capi-nntrainer-devel
+Developmental kit for Tizen Native NNTrainer API.
+
+%package -n capi-nntrainer-devel-static
+Summary:         Static library for Tizen Native API
+Group:           Multimedia/Framework
+Requires:        capi-nntrainer-devel=%{versin}-%{release}
```

Comment:
- Typo (versin)
- Style: Add spaces around = and version's -

Reply:
fixed. Thanks.